future/name-resolution/002-nr-host-synchronization/README.md

```diff
@@ -18,35 +20,37 @@ The significant things to note about this internet are that we have 2 machines o
 
 Avahi is a program which uses [multicast](../../../chapters/glossary.md#multicast) to perform name resolution on local networks with minimal configuration. If you check the [Dockerfile](./Dockerfile) for this chapter, you'll see that we added a new software, `avahi-utils`.
 
-You'll recall, in [chapter 1](../001-nr-getting-started/README.md#how-does-your-computer-know-where-to-go-to-resolve-a-name), we took a look at the contents of `/etc/nsswitch.conf`. We saw that the `hosts` line directed the computer how to resolve a name. It started with looking at the `files` on the system, e.g. `/etc/hosts`, and if it didn't find the name there, then it could use `dns` if it were configured.
+As you might recall, in [chapter 1](../001-nr-getting-started/README.md#how-does-your-computer-know-where-to-go-to-resolve-a-name), we took a look at the contents of `/etc/nsswitch.conf`. We saw that the `hosts` line provided instructions for how to resolve a name. It started with looking at the `files` on the system (i.e. `/etc/hosts`), and, if it didn't find the name there, then it could use `dns` (if it were configured).
+
+Let's start off by taking a quick look at the `/etc/nsswitch.conf` file on our machines now.
 
-Once you've `restart`ed for this chapter, `hopon` a host and `cat /etc/nsswitch.conf`. We'll see that `avahi-utils` added a couple new entries into that `hosts` line to direct name resolution requests for itself.
+Once you've `restart`ed for this chapter, `hopon host-c` and `cat /etc/nsswitch.conf`. This file suddenly looks a little different! The reason for this is that when we installed `avahi-utils`, it added a couple new entries into that `hosts` line to direct name resolution requests for itself.
 
 ```bash
 hosts:          files mdns4_minimal [NOTFOUND=return] dns
 ```
 
-Let's look at what each of these is doing; a couple of them will be review:
+Let's look at what each of these entries is doing (a couple of them will be review from chapter 1):
 
-* `files`: Is there an entry for this hostname in a local file? In UNIX based systems that file would be `/etc/hosts`.
-* `mdns4_minimal`: can this name be resolved using a multicast resolver? This is specific to resolving hostnames in the local network.
-* `[NOTFOUND=return]`: if the hostname matches the TLD for `mdns4_minimal`, e.g. `.local`, but the hostname cannot be resolved, don't send this request out to the open internet. For example, if we requested `host-x.local`, which doesn't exist on our little internet, don't make an open internet request.
-* `dns`: We gotta outsource this request to the larger internet; check the `/etc/resolv.conf` file for where we should send our DNS queries.
+* `files`: Is there an entry for this hostname in a local file? In UNIX-based systems, that file would be `/etc/hosts`.
```

Comment:
I think we should edit this description to be a statement like the rest of the bullets